container/Dockerfile.tensorrt_llm

```diff
@@ -116,15 +122,61 @@ COPY deploy/dynamo/sdk /workspace/deploy/dynamo/sdk
 
 # Install uv, create virtualenv for general use, and build dynamo wheel
 COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+### VIRTUAL ENVIRONMENT SETUP ###
+RUN mkdir /opt/dynamo && \
+    uv venv /opt/dynamo/venv --python 3.12
+
+###################################
+####### WHEEL BUILD STAGE #########
+###################################
+
+# Build the wheel in the manylinux environment
+FROM ${MANYLINUX_IMAGE} AS wheel_builder
+ARG RELEASE_BUILD
+ARG CARGO_BUILD_JOBS
+# Set CARGO_BUILD_JOBS to 16 if not provided
+# This is to prevent cargo from building $(nproc) jobs in parallel,
+# which might exceed the number of opened files limit.
+ENV CARGO_BUILD_JOBS=${CARGO_BUILD_JOBS:-16}
+
+WORKDIR /workspace
+
+RUN yum install -y protobuf-compiler \
+    && yum clean all \
+    && rm -rf /var/cache/yum
+
+COPY --from=build /workspace /workspace
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH \
+    CARGO_TARGET_DIR=/workspace/target
+
+COPY --from=build $RUSTUP_HOME $RUSTUP_HOME
+COPY --from=build $CARGO_HOME $CARGO_HOME
+
+# Create virtualenv and build dynamo wheel
 RUN mkdir /opt/dynamo && \
     uv venv /opt/dynamo/venv --python 3.12 && \
     source /opt/dynamo/venv/bin/activate && \
     cd /workspace/lib/bindings/python && \
-    uv build --wheel --out-dir /workspace/dist && \
-    uv pip install /workspace/dist/ai_dynamo_runtime*cp312*.whl && \
+    uv build --wheel --out-dir /workspace/dist --python 3.12 && \
+    if [ "$RELEASE_BUILD" = "true" ]; then \
+        uv build --wheel --out-dir /workspace/dist --python 3.11 && \
+        uv build --wheel --out-dir /workspace/dist --python 3.10; \
+    fi && \
     cd /workspace && \
-    uv build --wheel --out-dir /workspace/dist && \
-    uv pip install /workspace/dist/ai_dynamo*any.whl
+    uv build --wheel --out-dir /workspace/dist
+
+########################################
+########## Development Image ###########
+########################################
+
+FROM build AS dev
```

Comment:
should we just move the build stage to here and call it dev?

I don't think it is building anything if I understand correctly?